Changelog.md

```diff
@@ -11,6 +11,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Code generator: Fix missing dependency tracking for abstract contracts.
```

Comment:
Is this a IR-only change?

Reply:
Nope, it's also for normal output.

Reply:
It's both IR and old codegen, same issue.